generic/tests/cfg/arm_kvm_unit_tests.cfg

```diff
@@ -0,0 +1,11 @@
+- arm_kvm_unit_tests:
+    virt_test_type = qemu libvirt
+    type = arm_kvm_unit_tests
+    only aarch64
+    vms = ""
+    start_vm = no
+    repo_url = https://gitlab.com/kvm-unit-tests/kvm-unit-tests
+    variants sub_type:
```

Comment:
Do we need to split it? Just like `xfstest`, we can directly run this collection `./run_tests.sh`, and then print out the error cases.

Reply:
Actually I only want to run sub set in a sub test, this is more controllable for fine-grained tests. But I will use `--group` for the testing.

Reply:
Hum, `--group` always returns `0`, I need to consider again.

Reply:
Finally, I want to keep this, it's more controllable for subgroups.